src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -119,7 +119,6 @@
     <Compile Include="System\Xml\Core\XmlSubtreeReader.cs" />
     <Compile Include="System\Xml\Core\XmlSubtreeReaderAsync.cs" />
     <Compile Include="System\Xml\Core\XmlTextEncoder.cs" />
-    <Compile Include="System\Xml\Core\BinaryCompatibility.cs" />
```

Comment:
Looks like there is no contract for System.Threading.Tasks.Extensions. Keeping the project reference.